CryptoTechReminderSystem/UseCase/RemindProjectManagers.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using CryptoTechReminderSystem.Boundary;
+
+namespace CryptoTechReminderSystem.UseCase
+{
+    public class RemindProjectManagers
+    {
+        private readonly IGetLateBillablePeople _getProjectManagersWithOpenTimeEntries;
+        private readonly ISendReminder _sendReminder;
+
+        private bool _testing;
+        public RemindProjectManagers(IGetLateBillablePeople getProjectManagersWithOpenTimeEntries, ISendReminder sendReminder)
+        {
+            _getProjectManagersWithOpenTimeEntries = getProjectManagersWithOpenTimeEntries;
+            _sendReminder = sendReminder;
+
+            _testing = true;
```

Comment:
Feature flag (see private method below).  Once confident in production, will submit a PR to change to false.

Reply:
I'd recommend removing it once confident in production ;) otherwise, it's just dead code

Reply:
Good point!